src/components/hosts/HostRowDetail.tsx

```diff
@@ -66,30 +75,41 @@ const diskColumns = [
 
 const diskRowKey = ({ rowData }: ExtraParamsType) => rowData?.key;
 
+const DisksTableRowWrapper = (props: RowWrapperProps) => (
+  <RowWrapper {...props} data-testid={`disk-row:${props.row?.key}`} />
+);
+
 const DisksTable: React.FC<DisksTableProps> = ({ cluster, host, disks, installationDiskPath }) => {
   const isEditable = canEditDisks(cluster.status, host.status);
   const rows = disks
     .sort((diskA, diskB) => diskA.name?.localeCompare(diskB.name || '') || 0)
     .map((disk) => ({
       cells: [
-        { title: disk.bootable ? `${disk.name} (bootable)` : disk.name },
+        {
+          title: (
+            <span data-testid={'disk-name'}>
+              {disk.bootable ? `${disk.name} (bootable)` : disk.name}
+            </span>
+          ),
+        },
         {
           title: (
             <DiskRole
+              testId={'disk-role'}
               host={host}
               disk={disk}
               installationDiskPath={installationDiskPath}
               isEditable={isEditable}
             />
           ),
         },
-        { title: <DiskLimitations disk={disk} /> },
-        disk.driveType,
-        fileSize(disk.sizeBytes || 0, 2, 'si'),
-        disk.serial,
+        { title: <DiskLimitations testId={'disk-limitations'} disk={disk} /> },
+        { title: <span data-testid={'drive-type'}>{disk.driveType}</span> },
```

Comment:
Did you try using `IRow`'s `props` to pass the testid?
```suggestion
        {
          title: disk.driveType,
          props: { 'data-testid': 'drive-type' }
        },
```

Reply:
Interesting... I wasn't aware that there was such API. Can you help me with a link to the docs for that API? With exception of the table's `rowWrapper` on PF's Table component I couldn't find another way to achieve this.

Reply:
Thanks, I'll adjust the code in order to do just that.